branchinfo.txt

```diff
@@ -4,5 +4,5 @@
 MAJOR_VERSION=1
 MINOR_VERSION=0
 PATCH_VERSION=0
-RELEASE_SUFFIX=preview1
-CHANNEL=beta
+RELEASE_SUFFIX=preview2
+CHANNEL=preview
```

Comment:
I am going to upload the latest `beta` channel binaries to the `preview` channel so that we have a good stage0.